eq/channel.cpp

```diff
@@ -804,8 +799,11 @@ void Channel::declareRegion( const PixelViewport& region )
 
 PixelViewport Channel::getRegion() const
 {
-    PixelViewport region;
-    for( size_t i = 0; i < _impl->regions.size(); ++i )
+    if( _impl->regions.empty( ))
+        return PixelViewport();
```

Comment:
The real bug is that merge(invalid, empty) should yield empty and not invalid. Fix will come soon.